source/computer/fedora-setup.rst

```diff
@@ -176,7 +171,7 @@ Intel 软件开发工具包
 ^^^^^^^^^^^^^^^^^^^^
 
 `Intel oneAPI Toolkits <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
-是英特尔最新的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc`` 和 ``ifort`` 命令）。
+是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc``\，``i++`` 和 ``ifort`` 命令）。
```

Comment:
```suggestion
是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc``\ ，``i++`` 和 ``ifort`` 命令）。
```

Reply:
I don't see the `i++` command although I have installed the OneAPI base and HPC HPC toolkits.

Reply:
Hmm, so `icc` can compile C and C++ files?

Reply:
I am testing it.

Reply:
https://scc.ustc.edu.cn/zlsc/sugon/intel/compiler_c/main_cls/bldaps_cls/common/bldaps_invoke_cl_comm.htm

It seems it provides icc and icpc, and both are the same.

Reply:
I can use both `i++` and `icpc` to complile the hello-world.c++

```C++
 #include <iostream>
 
 int main() {
      std::cout << "Hello World!";
      return 0;
 }
```
```

Reply:
Did you mean `icc` or `icc`?

Reply:
I don't have `i++` and I think Intel OneAPI don't have such command.

Reply:
OK. I think we should only mention `icc` and `ifort`. I believe most C++ users are not our target audiences.

Reply:
I think so.

Reply:
See https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html. I installed both base and HPC toolkits. So I should have C and C++.